apps/juxtaposition-ui/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+	"compilerOptions": {
+		"strict": true,
+		"sourceMap": true,
+		"resolveJsonModule": true,
+		"module": "ES2022",
```

Comment:
Is this safe? I know that the target was set to es2022 before you touched this codebase, but thinking back on it won't this cause issues when making the jump to TypeScript for the client stuff? The Wii U and 3DS definitely can't handle any modern transpilation

Reply:
There is no frontend TS transpilation in juxt-ui yet. We'll cross that bridge when we get there.

Also you can specify a tsconfig.json in a subfolder, which is what we can use to separate frontend and backend code.

Reply:
I figured we'd get to it later, this was one of those "not an issue, just a discussion question" comments. I agree with crossing this bridge later, it was just something that caught my eye and wanted to ask before I forgot 👍